tests/misra/test_misra.sh

```diff
@@ -27,8 +27,13 @@ cd $PANDA_DIR
 scons -j8
 
 cppcheck() {
-  build_dir=/tmp/cppcheck_build
-  mkdir -p $build_dir
+  build_dir=/tmp/cppcheck_build_$7
```

Comment:
let's make a common root for them
```suggestion
  build_dir=/tmp/cppcheck_build/$7
```